### ISSUE TITLE
Google cloud picker sample doesn't work for type '*'

### ISSUE THREAD

**Issue by RByers** [PaulKinlan/WebIntents#143]
Modify the example at 'http://examples.webintents.org/intents/pick/pick.html' to do 'new Intent("http://webintents.org/pick", "_") instead of type "image/_" and try to use the google clode image picker.  You get a completely empty window.  

Ideally in this case we'd use the Google Picker API to show multiple file types (not just images), but at a minimum we should get at least as much as we get by asking for image/*.

**Comment by PaulKinlan** [comment]
Yes, agree.  This needs to be more comprehensive than it is now.  One thing I wasn't sure of was what to do with Docs files such as spreadsheets which we don't have our own defined mime type.

**Comment by PaulKinlan** [comment]
Another note is that the UI will open currently only look for image in it's test of what to work out what to do.
